toqito/matrix_props/is_circulant.py

```diff
@@ -25,12 +25,15 @@ def is_circulant(mat: np.ndarray) -> bool:
 
     As can be seen, this matrix is circulant. We can verify this in
     :code:`|toqito⟩` as
+    .. jupyter-execute::
+
+     import numpy as np
+     from toqito.matrix_props import is_circulant
+
+     mat = np.array([[4, 1, 2, 3], [3, 4, 1, 2], [2, 3, 4, 1], [1, 2, 3, 4]])
+
+     is_circulant(mat)
```

Comment:
Something has gone wrong with the output of this code block. I think maybe we just need to add a blank line before the code block.